marbl/handler.go

```diff
@@ -65,7 +69,7 @@ func (h *Handler) streamLogs(conn *websocket.Conn) {
 		log.Errorf("logstream: failed to create ID: %v", err)
 		return
 	}
-	framec := make(chan []byte, 10)
+	framec := make(chan []byte, 16384)
```

Comment:
is there some significance to 16384?

Reply:
Just 2^14, should this be parameterized?